tests/update_workflow_test.go

```diff
@@ -253,27 +248,24 @@ func (s *UpdateWorkflowSuite) TestUpdateWorkflow_EmptySpeculativeWorkflowTask_Ac
 
 func (s *UpdateWorkflowSuite) TestUpdateWorkflow_NotEmptySpeculativeWorkflowTask_AcceptComplete() {
 	testCases := []struct {
-		Name     string
-		UseRunID bool
+		Name       string
+		ApplyRunID func(tv *testvars.TestVars, runID string) *testvars.TestVars
 	}{
 		{
-			Name:     "with RunID",
-			UseRunID: true,
+			Name:       "with RunID",
+			ApplyRunID: func(tv *testvars.TestVars, runID string) *testvars.TestVars { return tv.WithRunID(runID) },
 		},
 		{
-			Name:     "without RunID",
-			UseRunID: false,
+			Name:       "without RunID",
+			ApplyRunID: func(tv *testvars.TestVars, _ string) *testvars.TestVars { return tv },
```

Comment:
I like the `true/false` approach much better. It means I can read the test case inputs very quickly/easily. I don't think the extra `if` is an issue.

Reply:
ok, let it be `bool` again.